src/View.php

```diff
@@ -613,16 +613,38 @@ public function jsUrl($page = [])
      */
     public function url($page = [])
     {
-        return $this->getApp()->url($page, false, $this->_getStickyArgs());
+        return $this->getApp()->url($page, false, array_merge($this->getRunningCallbackArgs(false, is_array($page) ? $page : []), $this->getStickyArgs()));
+    }
+
+    protected function getRunningCallbackArgs(bool $isTerminated, array $page): array
+    {
+        $args = [];
+        foreach ($this->elements as $v) {
+            if ($v instanceof Callback) { // @phpstan-ignore-line
+                if (($page[Callback::URL_QUERY_TARGET] ?? null) === $v->getUrlTrigger()) {
+                    $isTerminated = true;
+                }
+
+                if ($isTerminated && $v->isTriggered() && $v->canTrigger()) {
```

Comment:
@ibelar does `$v->canTrigger()` make sense here and any idea how to fix the remaining failing Behat tests?

Reply:
> @ibelar does `$v->canTrigger()` make sense here and any idea how to fix the remaining failing Behat tests?

Yes, canTrigger method will check for a reload. Normally, JsCallback should not run during a reload.

Reply:
I see why the test is failing. ViewTester reload is missing Callback arguments. Thus when reload is call, the callback method is not run. 

I think the reason is that ViewTester is added directly to Modal, while the callback is added to Button

`$button->on('click', $vp1Modal->show());`

Resulting in App -> Button -> Cb
                             -> Modal -> ViewTester (reload)

Thus the callback is not part of the Modal render tree... missing its arguments when reload URL is rendered.

Reply:
> Thus the callback is not part of the Modal render tree...

any idea how to solve it?

Reply:
@ibelar the issue is here: https://github.com/atk4/ui/blob/8014b6c1cb5beb103f337af8ace5ac350f73ce19/src/JsReload.php#L58

1. in https://github.com/atk4/ui/pull/1646 you have removed `mergeStickyArgsFromChildView` which allowed to delegate sticky args from it's children.
2. in this PR, sticky args are collected from the render tree, but the URL must be obtained thru terminating callback
3. here, hovewer the URL is obtained thru `$this->view` which is placed correctly within the Modal render tree, but the Modal is not delegating it's callback sticky args as `mergeStickyArgsFromChildView` was removed

I belive, `mergeStickyArgsFromChildView` is needed. Or how else should the `$this->view->jsUrl(...` in JsReload work?